### DISCUSSION TITLE
Sharing ideas to improve the software

### DISCUSSION THREAD

**Discussion by frett27** [barrelorgandiscovery/virtualbookplayer#2] [Ideas]
Sharing ideas to improve the project

**Comment by generalgruntin** [comment]
Is there a way to get this as a stand alone image for a raspberry pi? I'm building a MIDI step motor organ and this would be perfect.

**Reply by frett27** [reply]
I've never tested, but, this work on linux, and should work on rpi, (as it is linux), you can compile it yourself, and it quite simple, using rustup.
the steps are the following :
- install rustup on the rpi, 
- clone the repository (git clone ... )
- execute "cargo build" from the repository root

**Reply by frett27** [reply]
i had a look at the availability to be able to run compilation on rpi, and provide "out of the shelve" rpi binary, but there are not rpi plateform for compilation available on github :
https://docs.github.com/fr/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
